### ISSUE TITLE
Support multiple keys with `safe gen`

### ISSUE THREAD

**Issue by geofffranks** [starkandwayne/safe#52]
If there's a backwards-compatible way to support `safe gen [len] path key1 key2 ... keyN`, it would be great, so that we can make use of that similarly to `safe set path key1 key2 ... keyN`.

**Comment by jhunt** [comment]
We can always just assume that if the first path is numeric (`/^[0-9]$/`), it's a length, otherwise it's the start of a set of paths to gen for?

**Comment by jhunt** [comment]
Once #73 is merged, we can relegate length to an option, and if the path looks like an integer, fall back to the existing logic.  Then we will be able to do this:

```
safe gen -l 64 secret/path/to/wherever username password firstname lastname ssn phone email
```

**Comment by thomasmmitchell** [comment]
I believe the fix for this was included in #75 . Can we verify and close this ticket?

**Comment by jhunt** [comment]
Verified.

```
→  safe gen secret/stuff:{username,password,pin,token}
→  safe tree --keys secret
.
└── secret
    └── stuff
        ├── :password
        ├── :pin
        ├── :token
        └── :username

→  safe read secret/stuff
--- # secret/stuff
password: QjfrGk7tK6J6LtOA4hp7W9FVPs7iZpj3r5qyquupqE6SaV81fbQeZPguC2A2S8mG
pin:      LyJVKrGUMh4uZ8pWLv6vMl030sAmJtGgSdhcXrbyhRYHiDIYCFH4zj11UAHi0j6x
token:    Ynm4wwGmY7a44lj3GY06tR8edgO0jWc6rrLCt1dtzLc1gyv8SdfTLPtYpxyqPmrt
username: nHbWej7S44OD6NfaZ3qgRcgIHQ4gBFLlos3bEFWZ6XMLMhEqBH5kGcPabAGUsG0b
```
